github/compliance/report.py

```diff
@@ -586,21 +587,24 @@ def close_issues_for_absent_labels(
         state='open',
     )
 
-    def ocm_resource_label(issue: github3.issues.Issue) -> str:
+    digest_label_prefix = github.compliance.issue.issue_type_digest_label_prefix(issue_type)
```

Comment:
prefix should depend on target (type). if we do not know the target type, then we will have to search for all known prefixes instead